magprime/models.py

```diff
@@ -0,0 +1,20 @@
+from magprime import *
```

Comment:
I moved the model code into its own module, since we might very well add more models later.

Side note: I didn't have any trouble adding new models in a plugin, which @kitsuta mentioned was throwing all kinds of errors when she tried to do it for Anthrocon.  I'm guessing one of the million changes/fixes/additions which were made between then and now made that problem go away.